contract-tests/images/applications/mysql-connector/mysql_connector_server.py

```diff
@@ -0,0 +1,90 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+import atexit
+import os
+from http.server import BaseHTTPRequestHandler, ThreadingHTTPServer
+from threading import Thread
+from typing import Tuple
+
+import mysql.connector as mysql
+
+from typing_extensions import override
+
+_PREPARE_DB: str = "prepare_db"
+_SELECT: str = "select"
+_CREATE_DATABASE: str = "create_database"
+_DROP_TABLE: str = "drop_table"
+_FAULT: str = "fault"
+_PORT: int = 8080
+
+_DB_HOST = os.getenv("DB_HOST")
+_DB_USER = os.getenv("DB_USER")
+_DB_PASS = os.getenv("DB_PASS")
+_DB_NAME = os.getenv("DB_NAME")
+
+
+class RequestHandler(BaseHTTPRequestHandler):
+    @override
+    # pylint: disable=invalid-name
+    def do_GET(self):
+        status_code: int = 200
+        conn = mysql.connect(host=_DB_HOST, user=_DB_USER, password=_DB_PASS, database=_DB_NAME)
+        conn.autocommit = True  # CREATE DATABASE cannot run in a transaction block
+        if self.in_path(_PREPARE_DB):
+            cur = conn.cursor()
+            cur.execute("CREATE TABLE employee (id int, name varchar(255))")
+            cur.execute("INSERT INTO employee (id, name) values (1, 'A')")
+            cur.close()
+            status_code = 200
```

Comment:
Can be discussed looking for a better way than adding an endpoint 
problem is on tests I would have to change a lot in the base tests if all of these operations would be done under SELECT endpoint.

So I call Prepare before select and then clear the mock collector.

Reply:
Please explain more about why it does not align with psycopg2/pymysql tests. If it is necessary, please also update the psycopg2/pymysql tests.

Reply:
I'm creating mysql-connector-driver with additional operation to test which is SELECT.

To be able to test SELECT alone, I have added this prepare db to add data into the DB before firing the SELECT query.

for Other DB drivers I. can once I get agreement on this approach also I think its ok to be part of other PR as well.

Reply:
Addressed all DB Drivers to have the same test.